installer/test/phx_new_umbrella_test.exs

```diff
@@ -48,7 +48,7 @@ defmodule Mix.Tasks.Phx.New.UmbrellaTest do
       # Phoenix.LiveView.HTMLFormatter
       if Version.match?(System.version(), ">= 1.13.4") do
         assert_file root_path(@app, "mix.exs"), fn file ->
-          assert file =~ "{:phoenix_live_view, \">= 0.0.0\"}"
+          assert file =~ ~r/defp deps do\n.*\[\n.*\n.*{:phoenix_live_view, \">= 0.0.0\"}/
```

Comment:
@josevalim both are fine, it's up to you if should be more specific of if just checking `{:phoenix_live_view, \">= 0.0.0\"}"` is enough

Reply:
I think just checking the tuple is enough!